lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.0.0-SNAPSHOT",
+  "version": "3.0.0-next.202308311221",
```

Comment:
I don't think we need to specify a pre-release string here.
I don't believe it will hurt to keep it though.

Reply:
+1 since the version should get automatically updated by our deploy workflow to be `3.0.0-next.<timestamp>` at time of publish